reStream.sh

```diff
@@ -87,6 +93,11 @@ ssh_cmd() {
     ssh -o ConnectTimeout=1 -o PasswordAuthentication=no "root@$remarkable" "$@"
 }
 
+# SSH_CONNECTION is a variable on reMarkable => ssh '' instead of ssh ""
+remarkable_ip() {
+    ssh -o ConnectTimeout=1 -o PasswordAuthentication=no "root@$remarkable" 'echo $SSH_CONNECTION' | cut -d\  -f3
```

Comment:
```suggestion
    ssh_cmd 'echo $SSH_CONNECTION' | cut -d\  -f3
```

Reply:
Just wondering, will this always return a usable IP address?

Reply:
From `ssh` man page
> SSH_CONNECTION Identifies the client and server ends of the connection. The variable contains four space-separated values: client IP address, client port number, server IP address, and server port number.

Reply:
I get a linting error with `ssh_cmd 'echo $SSH_CONNECTION' | cut -d\  -f3`, but work as expected. Tell me if for you is ok!